packages/mephisto-task/README.md

```diff
@@ -46,56 +58,62 @@ function MyApp() {
 The `useMephisoTask` React hook exposes the following fields:
 
 ### `taskConfig`
-A config object passed from the backend.
+An arbitrary task-specific config object passed from the backend.
 
 Example:
 ```json
 {
-    block_mobile: true
-    frame_height: 650
-    get_task_feedback: false
-    num_subtasks: 2
-    question: "Who would you prefer to talk to for a long conversation?"
-    task_description: "Placeholder Task Description - Javascript failed to load"
+    "block_mobile": true,
+    "frame_height": 650,
+    "get_task_feedback": false,
+    "num_subtasks": 2,
+    "question": "Who would you prefer to talk to for a long conversation?",
+    "task_description": "Pick the AI that you find more engaging."
 }
 ```
-
-### `providerWorkerId`
-### `mephistoWorkerId`
 ### `agentId`
+
+An `agentId ` represents a worker working on a task. A single task could have multiple `agentId`s, for example in the case of conversational chat tasks. An agent could also be a model-in-the-loop instead of a worker.
+
+Usually you'll want to use `agentId` in your task code to represent workers as opposed to `mephistoWorkerId` and `providerWorkerId` which are reserved for more advanced usages. 
+
 ### `assignmentId`
 
+When a worker accepts a task, an `assignmentId` is created to represent the pairing between the worker and the task.
+
 ### `initialTaskData`
 
-The initial data to populate your task with. The crowdworker will use this data to perform their action.
+The initial data to populate your task with. The worker will use this data to perform their action.
```

Comment:
Useful to note where this comes from on the backend - it's the `InitializationData` object that the TaskRunner returns in `get_data_for_assignment`, which comes from the set of `InitializationData` objects returned by the blueprint's `get_initialization_data` method.